src/includer/parsers.ts

```diff
@@ -110,6 +110,15 @@ function tagsFromSpec(spec: OpenAPISpec): Map<string, Tag> {
         }
     }
 
+    parsed.forEach((tag: Tag) => {
+        if (tag['x-navtitle']) {
+            tag.name = tag['x-navtitle'];
```

Comment:
I have my reservations about the fact that we have zero distinction between a `Tag` that originates from an OpenAPI spec and a `Tag` that gets actually used when generating the ToC. I consider having two distinct entity types a better solution, where the includer-specific entity would bear no references to spec-specific vendor extensions such as `x-navtitle` or `x-slug`. However, I do recognize that this is out of scope for this PR, since this would require a significant rewrite of these parsers/transformers.